src/hash_algorithms.jl

```diff
@@ -182,6 +182,12 @@ function update_hash!(hasher::BufferedHashState, obj, context)
     return hasher
 end
 
+function update_hash!(hasher::BufferedHashState, obj)
```

Comment:
Bumping the hash version used to test BufferedHashState via a mockup buffer type, means this method needs to be implemented now. It needs to eventually be implemented anyways (see TODO comment for the method above this one), once the deprecated behavior for hash versions 1-3 are removed.